projects/observability/src/shared/components/utils/svg/svg-util.service.ts

```diff
@@ -150,13 +150,12 @@ export class SvgUtilService {
           this.truncateText(tspan.node()!, width);
         } else {
           // Move this word to next line
-
+          const y = textSelection.attr("y");
+          const dy = parseFloat(textSelection.attr("dy"));
           currentLine.pop();
           tspan.text(currentLine.join(' '));
           currentLine = [word];
-          tspan = appendTSpan()
-            .attr('dy', `${++lineNumber * 1.1}em`)
-            .text(word);
+          tspan = appendTSpan().attr("x", 0).attr("y", y).attr("dy", `${++lineNumber * 1.1 + dy}em`).text(word)
```

Comment:
nit: put `.<method>` on a new line

Reply:
oh, I thought if prettier doesn't complain I shall leave it that way. Updated